ext/JLD2JUDIExt.jl

```diff
@@ -10,7 +10,7 @@ JUDI.Geometry(x::JLD2.ReconstructedMutable{N, FN, NT}) where {N, FN, NT} = Geome
 function JUDI.tof32(x::JLD2.ReconstructedStatic{N, FN, NT}) where {N, FN, NT}
     # Drop "typed" signature
     reconstructT = Symbol(split(string(N), "{")[1])
-    return JUDI.tof32(eval(reconstructT)([getproperty(x, f) for f in FN]...))
+    return JUDI.tof32(getproperty(@__MODULE__, reconstructT)([getproperty(x, f) for f in FN]...))
```

Comment:
```suggestion
    return JUDI.tof32(getglobal(@__MODULE__, reconstructT)([getproperty(x, f) for f in FN]...))
```
(assuming you only support v1.9 or later)

Reply:
I believe there's a 1.6 compat, which is why I used get property. But yes, getglobal is preferred for 1.9+